src/ui/shared/requests/useQuotes.ts

```diff
@@ -148,6 +202,19 @@ export function useQuotes2({
           return updatedItem || item;
         });
       },
+      eventCodeToMessage: QUOTES_EVENT_CODE_TO_MESSAGE,
+      onError: ({ parsedError, rawEvent, requestUrl }) => {
+        handleQuoteError({
+          message: parsedError.message,
+          code: rawEvent.code,
+          backendMessage: rawEvent.message,
+          requestParams: Object.fromEntries(
```

Comment:
Form state can change before onError call, so requestParams used to pass correct form state into the handleQuoteError callback.